### PR TITLE
Expose API version descriptors for admins via API

### PR DESCRIPTION
**Pull request by matmair**
This PR adds a new endpoint to retrieve the API version texts and unit tests for it.

### CODE REVIEW

File: InvenTree/InvenTree/api.py [hunk 1]
```diff
@@ -57,6 +58,18 @@ def get(self, request, *args, **kwargs):
...
+class VersionTextView(APIView):
+    """Simple JSON endpoint for InvenTree version text."""
+
+    permission_classes = [
+        permissions.IsAdminUser,
+    ]
+
+    def get(self, request, *args, **kwargs):
+        """Return information about the InvenTree server."""
+        return JsonResponse(inventreeApiText())
```
Comment (wolflu05) [inline InvenTree/InvenTree/api.py:70]: Question: I asked me this a while ago, shouldn't I use the Response from DRF, that automatically is a json response? Or should I use the response from Django is such situations?

Comment (matmair) [reply InvenTree/InvenTree/api.py:70]: You can use either

**Comment by wolflu05** [comment]
Interesting, Shouldn't the coverage normally be 100% for strings that are defined on the global scope, if that file is imported somewhere?

**Comment by matmair** [comment]
Yes @wolflu05 but there seems to be a long known problem/behaviour that only the first line is reported as covered - still thinking about how to tackle that if this solution is not working

**Comment by wolflu05** [comment]
Ah thats bad. But this no cover comment in the first line doesn't look great, maybe we could deactivate coverage for that file, and factor everything else out?

**Comment by matmair** [comment]
> Ah thats bad. But this no cover comment in the first line doesn't look great, maybe we could deactivate coverage for that file, and factor everything else out?

I find that suggestion by far less clean than excluding certain lines.

**Review by SchrodingersGat — APPROVED** [review]

**Comment by matmair** [comment]
I have removed the coverage things altogether, some larger things should be investigated and solved consistently

**Comment by matmair** [comment]
@SchrodingersGat this is ready for review and merge

**Comment by SchrodingersGat** [comment]
Thanks @matmair , LGTM

**Pull request merged**